From f2eaa1c40110acb0dbdd716fed3254947d5c03a7 Mon Sep 17 00:00:00 2001
From: "Cao,Bing Bu" <bingbu.cao@intel.com>
Date: Fri, 31 Oct 2014 13:43:17 +0000
Subject: [PATCH 114/117] CHERRY PICK: Revert "[Revert Me]atomisp2: tune the
 dvs2_bq setting for IRDA ECS7"

This reverts commit a270fd51cb4c6edbffa4907a287b1483e72bdde5.
Revert this temporary patch to fix the SDV video corrupt issue.

Change-Id: Ic2fe252672db878289ecbf83013c168a12c8cefb
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-5545
Tracked-On: https://bz01p-vied.ir.intel.com/show_bug.cgi?id=4319
Signed-off-by: Cao,Bing Bu <bingbu.cao@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/26426
Reviewed-by: Jing Tao <jing.tao@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../pci/atomisp2/atomisp_driver/atomisp_cmd.c      | 25 ----------------------
 1 file changed, 25 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_cmd.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_cmd.c
index f4b94b2..398d9b5 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_cmd.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_cmd.c
@@ -2355,20 +2355,6 @@ int atomisp_get_dvs2_bq_resolutions(struct atomisp_sub_device *asd,
 				bq_res->gdc_shift_bq.width_bq = 4 / 2;
 				bq_res->gdc_shift_bq.height_bq = 4 / 2;
 			} else {
-/*
- * FIXME!
- * The DVS Parameter setting rule is still not settled down at present which
- * makes no general fomular which could fit for all the senarios.
- *
- * Previous parameter configuration formual was only tuned for Saltbay which
- * works well. However currently for IRDA it needs a new formular which is
- * still got by experiment.
- *
- * This issue is tracked by BZ 4358 in high priority to get from Firmware/DVS
- * Library team to give a clear general rule on how the parameter is
- * configured.
- */
-#ifndef CONFIG_GMIN_INTEL_MID
 				/*
 				 * spatial filter shift and more left padding in SDV
 				 * case, the left padding is
@@ -2391,17 +2377,6 @@ int atomisp_get_dvs2_bq_resolutions(struct atomisp_sub_device *asd,
 					pipe_cfg->bayer_ds_out_res.width /
 					input_config->effective_res.width + 1) /
 					2;
-#else
-				w_padding =
-					roundup(input_config->effective_res.width + 12, 128) -
-					input_config->effective_res.width;
-
-				bq_res->gdc_shift_bq.width_bq = 4 / 2 +
-					(w_padding *
-					pipe_cfg->bayer_ds_out_res.width /
-					input_config->effective_res.width + 1) /
-					2;
-#endif
 				bq_res->gdc_shift_bq.height_bq = 4 / 2;
 			}
 		} else {
-- 
1.9.3

